learn/on-premise/observability/logs-aggregation/kibana.md

```diff
@@ -52,6 +52,20 @@ Kibana is purely used for the logs generated by the Platform. Application logs g
 
     ![Kibana Discover Page](/learn/assets/wme-setup/wme-observability/kibana/kibana-discover-open.png)
 
+    **UseCaseOne**: Accessing Specific MicroService log( Developer-Cloud) from Kibana.
```

Comment:
Add this end of the doc. not in the middle.
Heading can be like Usecases. In that numbers/bullets..
Usecases
1. Accessing Specific MicroService Logs.
    There are micro services like login, developer-cloud.. write all micro service names.
    screen shot is wrong. it should be with tag not with log file name.
    
2.Access All MicroService Logs
   I didnt see dashboard for this. I assume u seen it in one of the wme setup

Reply:
2) I have seen Dashboard in the demo-setup and also in our exsi setup, based on that i have added

Reply:
Added the tag.keyword filter made the changes